game/piece.py

```diff
@@ -49,3 +63,43 @@ def handle_event(self, event):
             pygame.mouse.set_cursor(pygame.SYSTEM_CURSOR_HAND)
             return True
         return False
+    
```

Comment:
```suggestion
    
```Can you please remove the testing code when I merge it to the main branch?

Reply:
Wait sorry I did not mean to make a sugestion with code here, just try to remove the testing code here...